docs/tutorials/particle-filtering.md

```diff
@@ -0,0 +1,3 @@
+# Primer on Particle Filtering with Beluga
+
+Coming soon!
```

Comment:
@Alondruck I left this placeholder for the one-dimensional particle filter tutorial (#340).